packit/status.py

```diff
@@ -136,6 +137,29 @@ def get_builds(self,) -> Dict:
         builds = {b["nvr"].rsplit(".", 1)[1]: b["nvr"] for b in reversed(builds_l)}
         return builds
 
+    def get_copr_builds(self) -> List:
+        """
+        Get the copr builds of this project done by packit.
+        :return: list of builds
+        """
+        client = CoprClient.create_from_config_file()
+        package_name = self.package_config.downstream_package_name
+
+        projects = [
+            project.name
+            for project in reversed(client.project_proxy.get_list(ownername="packit"))
+            if package_name in project.name
+        ][:5]
+
+        builds: List = []
+        for project in projects:
+            builds += client.build_proxy.get_list(
+                ownername="packit", projectname=project, packagename=package_name
+            )
+
+        logger.debug("Copr builds fetched.")
+        return [(build.id, build.projectname, build.state) for build in builds]
```

Comment:
The list can be quite long since we take all builds for each project.
Maybe we can sort the list of builds based on 'ended_on' key and take let's say 5 latest builds?